irispy/iris_tools.py

```diff
@@ -66,18 +73,21 @@
 # Define whether IRIS WCS is 0 or 1 origin based.
 WCS_ORIGIN = 1
 
-def get_iris_response(pre_launch=False, response_file=None, response_version=None,
-                      force_download=False):
+
+def get_iris_response(time_obs=None, pre_launch=False, response_file=None, response_version=None,
```

Comment:
I think that if nothing is given then it should default to version 4. Such that you can run `get_iris_response()` it should run for version 4.

Reply:
Maybe another PR for this?